core/themes/seven/css/style.css

```diff
@@ -725,6 +731,12 @@ fieldset .fieldset-wrapper {
   border: .32em solid transparent;
   border-left-color: black;
 }
+.js[dir="rtl"] fieldset.collapsed .fieldset-legend a:before {
+  right: 0;
+  top: .75em;
+  border: .32em solid transparent;
```

Comment:
`top` rule can be removed entirely.

`border` rule is slightly redundant but it works to reset the border. Perhaps it should just be `border-left-color: transparent;`?

Reply:
fixed.